docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks.md

```diff
@@ -9,24 +9,74 @@ modeled in [Decision Model and Notation](https://www.omg.org/dmn/) (DMN).
+countryCode`). The expression is evaluated on activating the business rule task (or when an incident
+at the business rule task is resolved), after input mappings have been applied. The expression must
+result in a `string`.
 
-The differences between these task types are the visual representation (i.e. the task marker) and
-the semantics for the model.
+A business rule task must define the process variable name of the decision result as
+`resultVariable`. The result of the decision will be stored in this variable. The `resultVariable`
+is defined as a static value.
 
-When a process instance enters a business rule task, it creates a corresponding job and waits for
-its completion. A job worker should request jobs of this job type and process them. When the job is
-completed, the process instance continues.
+## Variable mappings
+
+By default, the variable defined by `resultVariable` is merged into the process instance. This
+behavior can be customized by defining an output mapping at the business rule task.
+
+All variables in scope of the business rule task are available to the decision engine when the
+decision is evaluated. Input mappings can be used to transform the variables into a format accepted
+by the decision.
 
 :::note
-Jobs for business rule tasks are not processed by Zeebe itself. To process them, you must provide a job worker.
+Input mappings are applied on activating the business rule task (or when an incident at the business
+rule task is resolved), before the decision evaluation. When an incident is resolved at the business
+rule task, then the input mappings are applied again before evaluating the decision. This can affect
+the result of the decision.
 :::
 
-## Defining a task
+For more information about this topic visit the documentation about [Input/output variable
+mappings](/components/concepts/variables.md#inputoutput-variable-mappings).
+
+## Alternative task implementation
+A business rule task does not have to evaluate a decision modeled with DMN. Instead, you can also
+use [job workers](/components/concepts/job-workers.md) to implement your business rule task.
+
+An alternative task implementation can be defined using the `zeebe:taskDefinition` extension element.
+
+Business rule tasks with an alternative task implementation behave exactly like [service
+tasks](/components/modeler/bpmn/service-tasks/service-tasks.md). The differences between these task
+types are the visual representation (i.e. the task marker) and the semantics for the model.
+
+When a process instance enters a business rule task with alternative task implementation, it creates
+a corresponding job and waits for its completion. A job worker should request jobs of this job type
+and process them. When the job is completed, the process instance continues.
+
+:::note
+Jobs for business rule tasks are not processed by Zeebe itself. To process them, you must provide a
+job worker.
+:::
```

Comment:
:wrench: We could remove this note because it is now clear from the context that the user must provide its own job worker.